orm.go

```diff
@@ -453,3 +469,19 @@ const (
 	TimeQuantumMonthDayHour     TimeQuantum = "MDH"
 	TimeQuantumYearMonthDayHour TimeQuantum = "YMDH"
 )
+
+type CacheType string
```

Comment:
That makes sense if the value is an int, and/or you want some type safety around it or to define methods on it, but I don't think it buys us anything here - users are going to use the constant values when declaring cache types - or they'll just be annoyed that they have to wrap a string in `pilosa.CacheType()` - so you'll have a very redundant looking line like:

```
CacheType: pilosa.CacheType("ranked"),
```